### PR TITLE
doc: update docs for default checksum strategy

### PR DESCRIPTION
**Pull request by SchrodingerZhu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Version Details of TiFlash Checksum

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/tiflash/pull/4010

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -65,8 +65,8 @@ delta_index_cache_size = 0
 
     ## DTFile 储存文件格式
     ## * format_version = 1 老旧文件格式，已废弃
-    ## * format_version = 2 默认文件格式
-    ## * format_version = 3 新文件格式，具有更完善的检验功能
+    ## * format_version = 2 v6.0.0 以前版本的默认文件格式
+    ## * format_version = 3 v6.0.0 及以后版本的默认文件格式，具有更完善的检验功能
     # format_version = 2
```
Comment (flowbehappy) [inline tiflash/tiflash-configuration.md:70]:
```suggestion
    # format_version = 3
```

File: tiflash/use-tiflash.md [hunk 2]
```diff
@@ -457,8 +457,9 @@ DTFile 存储在数据文件夹目录下的 stable 文件夹内。目前启用
 TiFlash 支持自动和手动进行数据校验：
 
 - 自动数据校验 （`storage.format_version` 配置项）：
-    - 默认使用 DTFile V2 版本校验机制。
-    - 如需开启 DTFile V3 版本校验机制，参见 [TiFlash 配置文件](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml)。
+    - v6.0.0 之后默认使用 DTFile V3 版本校验机制。
+    - v6.0.0 之前默认使用 DTFile V2 版本校验机制。
+    - 如需切换版本校验机制，参见 [TiFlash 配置文件](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml)。
```
Comment (flowbehappy) [inline tiflash/use-tiflash.md:462]:
```suggestion
    - 如需切换版本校验机制，参见 [TiFlash 配置文件](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml)。默认配置经过大量测试，不推荐修改。
```

**Comment by ran-huang** [comment]
@flowbehappy Please approve this PR if you have no further comments. Thanks.

**Review by flowbehappy — APPROVED** [review]

**Pull request merged**